orthoinference/src/main/java/org/reactome/orthoinference/SkipTests.java

```diff
@@ -0,0 +1,219 @@
+package org.reactome.orthoinference;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+
+import org.gk.model.ClassAttributeFollowingInstruction;
+import org.gk.model.GKInstance;
+import org.gk.model.InstanceUtilities;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+
+public class SkipTests {
+	
+	private static MySQLAdaptor dba;
+//	private static ArrayList<String> skipList = new ArrayList<String>();
+	static HashSet<String> skipList = new HashSet<String>();
+	
+	public static void setAdaptor(MySQLAdaptor dbAdaptor)
+	{
+		dba = dbAdaptor;
+	}
+	
+	// Skiplist was traditionally provided in a file, but since it's currently just 3 instances, I've just hard-coded them here.
```

Comment:
...but you _will_ restore the ability to read them from a file, right?

Reply:
Yes I'll code that in